src/test/java/io/github/bobocodebreskul/server/TomcatServerTest.java

```diff
@@ -25,6 +26,8 @@ class TomcatServerIntegrationTest {
 
   @BeforeEach
   void setUp() throws InterruptedException {
+    PropertiesConfiguration.loadProperties();
```

Comment:
This call was made just because NPE is thrown if loadProperties was not invoked, right?:)
Check the comments in PropertiesConfiguration file